docs/eventlisteners.md

```diff
@@ -430,618 +384,215 @@ metadata:
     service.beta.kubernetes.io/aws-load-balancer-backend-protocol: https
 ```
 
-**Note**: If there are any annotations attached to the service or the deployment, they will get overwritten 
-by the annotations available in the eventlistener.
-
-## Interceptors
```

Comment:
Should we at least have a link to the Interceptors page here ?

Reply:
Good catch! Added a "Specifying `Interceptors`" section with a short description and a link to the `Interceptors` page.